src/components/Button/Button.Props.ts

```diff
@@ -78,3 +90,13 @@ export enum ButtonType {
   command,
   icon
 }
+
+export enum IconColor {
+  neutralSecondary,
+  white
```

Comment:
I'm okay with this not being an enum, but do we want to support every color possibility (let people pass in the entire class) or limit it to just a few options?